src/main/scala/org/opennetworkinsight/dns/DNSWordCreation.scala

```diff
@@ -1,239 +1,95 @@
 package org.opennetworkinsight.dns
 
```

Comment:
proxy uses some of this same logic so it was moved to a shared DomainProcessor utitility